docs/authentication/configuration/session-options.mdx

```diff
@@ -1,5 +1,72 @@
+</Callout>
+
+### Inactivity timeout
+
+Denotes the duration after which a session will expire and the user will have to sign in again, if they haven't been active on your site. By default, this setting is disabled for all newly created instances. You can enable it and set your desired value from the **Sessions** settings in the Clerk Dashboard.
+
+<Images 
+width={3024}
+height={1652}
+src="/docs/images/authentication/session-options/inactivity-timeout.png"
+alt="The 'Sessions' configuration section in the Clerk Dashboard with a red outline around the 'Inactivity timeout' section. This option is toggled on, revealing an input for customizing the time duration until inactivity timeout. A red arrow is pointing to the 'Apply Changes' button."
+/>
+
+### Maximum lifetime
+
+Denotes the duration after which a session will expire and the user will have to sign in again, regardless of their activity on your site. By default, this setting is enabled with a default value of 7 days for all newly created instances. You can find this setting and change the value in the **Sessions** settings in the Clerk Dashboard.
+
+<Images 
+width={3024}
+height={1652}
+src="/docs/images/authentication/session-options/maximum-lifetime.png"
+alt="The 'Sessions' settings in the Clerk Dashboard with a red outline around the 'Maximum lifetime' section. This option is toggled on, revealing an input for customizing duration of a session lifetime."
+/>
+
+## Multi-session handling
+
+Multi-session applications allow users to be signed into more than one account at a time from the same browser. Users may switch the active account by opening the [`<UserButton />`](/docs/components/user/user-button) and selecting the desired account.
+
+To enable this feature in your application, you simply need to toggle on the **Mutli-session handling** option under the **Sessions** settings in the Clerk Dashboard.
+
+<Images 
+width={3024}
+height={1652}
+src="/docs/images/authentication/session-options/multisession-handling.png"
+alt="The 'Multi-session handling' section under the 'Sessions' settings in the Clerk Dashboard. A red arrow is pointing to the 'Multi-session handling' toggle, which is toggled on, and there is also a red arrow pointing to the 'Apply Changes' button."
+/>
+
+Learn more about working with multi-session applications with Clerk components and custom flows in our [detailed guide](/docs/custom-flows/multi-session-applications).
+
+## Customize session token
+
+Session tokens are JWTs that contain a set of standard claims required by Clerk, but there are times when you want to augment these tokens by providing additional claims of your own.
+
+To add custom claims to the session token, navigate to the **Sessions** settings in the Clerk Dashboard, and click on the **Edit** button in the **Customize session token** section. This will open a modal window where you can add claims and even use our handy shortcodes.
```

Comment:
```suggestion
To add custom claims to the session token, navigate to the [**Sessions**](https://dashboard.clerk.com/last-active?path=settings) settings in the Clerk Dashboard, and click on the **Edit** button in the **Customize session token** section. This will open a modal window where you can add claims and even use our handy shortcodes.
```